platform/security/core/security-core-impl/src/test/groovy/ddf/security/samlp/impl/LogoutMessageSpecTest.groovy

```diff
@@ -69,7 +69,7 @@ class LogoutMessageSpecTest extends Specification {
 
     def "build valid logout request"() {
         when:
-        LogoutRequest logoutRequest = logoutMessage.buildLogoutRequest(NAME_ID, ISSUER_ID)
+        LogoutRequest logoutRequest = logoutMessage.buildLogoutRequest(NAME_ID, ISSUER_ID, SESSION_INDEXES)
```

Comment:
❓ Shouldn't we also add an assertion for the session indices?

Reply:
Currently `SESSION_INDEXES` is an empty collection. I'll add a value and assert it.